### DISCUSSION TITLE
Share krita ai workflows

### DISCUSSION THREAD

**Discussion by locke12456** [Acly/krita-ai-diffusion#1332] [General]
Since the last update [v1.26.0], which introduced the customizable workflow feature, the potential of this tool has greatly expanded. It also allowed me to implement features I had previously wanted but hadn’t gotten around to, like layer diffusion and image-to-text via the WD14 tagger.

I’m curious if there are any similar places or discussion threads on this topic. If possible, I’d like to start the conversation by sharing two custom functions I’ve defined.
1. [layer diffusion workflow for krita ai](https://civitai.com/articles/8218/krita-ai-krita-ai-with-layer-diffusion)
2. [image to text](https://civitai.com/articles/8429/krita-ai-image-to-text)

**Comment by minsky91** [comment]
Come join our Discord user group at https://discord.com/invite/pWyzHfHHhU . We have a dedicated channel to share workflows for Krita, peope post there almost daily.

**Reply by patrickchamp** [reply]
Thanks! I was just going to suggest we find a spot to share workflows with each other.
